lib/v2/screens/dashboard/wallet_header.dart

```diff
@@ -12,6 +12,7 @@ import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/token_balances_
 
 import 'components/currency_info_card_widget.dart';
 
+// TODO(n13): Make this stateless
 class WalletHeader extends StatefulWidget {
   const WalletHeader({
     Key? key,
```

Comment:
I would remove this comma in the key params.

From 3 lines 
```
  const WalletHeader({
    Key? key,
  }) : super(key: key);
```
  
To 1 line

`  const WalletHeader({Key? key}) : super(key: key);`

Also change 3 lines override 

```
  @override
  State<StatefulWidget> createState() {
    return WalletHeaderState();
  }
```
  
  to 1 line too
  
```
  @override
  State<StatefulWidget> createState() => WalletHeaderState();
```

Reply:
This whole class will be changed to bloc so let's do that then (I agree)